src/errors.ts

```diff
@@ -18,69 +18,63 @@ interface RequestConfig {
   url?: string;
   method?: Method;
   baseURL?: string;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+
   headers?: any;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+
   params?: any;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+
   data?: any;
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
 interface Response<T = any> {
   data: T;
   status: number;
   statusText: string;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+
   headers: any;
   config: RequestConfig;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+
   request?: any;
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
 export interface HttpError<T = any> extends Error {
   config: RequestConfig;
   code?: string;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+
   request?: any;
   response?: Response<T>;
   isAxiosError: boolean;
   type: 'proxy' | 'oauth1' | 'oauth2';
   toJSON: () => Record<string, unknown>;
 }
 
-const cleanHeaders = (headers: Record<string, unknown>) =>
-  headers &&
+const cleanHeaders = (headers: Record<string, unknown>) => (headers &&
   'Authorization' in headers &&
```

Comment:
(Minor, no need to change) granted this was already not great, but can I suggest something more like this?
```ts
const cleanHeaders = (headers: Record<string, unknown>) => (
  headers && 'Authorization' in headers && typeof headers.Authorization === 'string' ?
  {
    ...headers,
    Authorization: `${
      headers.Authorization.startsWith('Bearer') ?
        '' :
        headers.Authorization.substring(0, 75)
    } ... ${headers.Authorization.substring(-20)}}`,
  } :
  headers
);
```